python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -399,17 +405,35 @@ def get_artifact(
 
         return orjson.loads(artifact_path.read_bytes())
 
+    def _format_exception_messages(self) -> str:
+        """Format the exception messages from the dbt CLI process."""
+        if not self._exception_messages:
+            return ""
+
+        exception_description = "\n".join(self._exception_messages)
+        return (
+            "\n\nThe above exception was caused by the following"
```

Comment:
I feel like it's possible for log messages to be produced at the "error" level without them explicitly being exceptions. e.g. if I'm remembering correctly, if a model errors, there will be an error-level log line saying: "3 executed, 10 skipped, 1 errored", which is not really the same shape as a python error that usually comes after that sort of string.

I think it's fine to just print "Errors parsed from dbt logs: " then the list of errors, just to make it explicit what's actually going on.

Reply:
I believe during testing that message isn't emitted at the error log level. But regardless, this is a good callout. I changed the message to explicitly note that we're only displaying the error messages, rather than implying that the failure was caused by the messages.